external/fv3fit/fv3fit/emulation/models/microphysics.py

```diff
@@ -67,11 +71,7 @@ def name(self):
 
     @property
     def output_variables(self):
-        return (
-            self.direct_out_variables
-            + list(self.residual_out_variables.keys())
-            + list(self.tendency_outputs.values())
-        )
+        return self.direct_out_variables + list(self.residual_out_variables.keys())
```

Comment:
I changed this property to only include the target variables that must be in the dataset. `.output_variables` is only used in `train_microphysics.py` anyway.